src/movement_controls.py

```diff
@@ -17,7 +22,7 @@
 #
 # for checking if a vehicle should take damage from an impact. as a percentage of their top acceleration
 
-DEFAULT_WORLD_SIZE = 750
+DEFAULT_WORLD_SIZE = constants.SCREEN_WIDTH
```

Comment:
Nice fix here, we can also make a different size for height as opposed to width?  Either in this PR or in https://trello.com/c/gynNhJrw/140-advanced-movement?

Reply:
Agreed, but I'll let advanced-movement take care of this. I just wanted to fix that bug where the car jumps to the other side of the screen